lib/birdsong/user.rb

```diff
@@ -57,6 +57,7 @@ def parse(json_user)
 
       @description = json_user["description"]
       @url = json_user["url"]
+      @url = "https://www.twitter.com/#{@username}" if @url.nil?
```

Comment:
Combine L59 and L60 into one line? Maybe that makes things to messy.

Reply:
I wanted to make it explicit that this is the backup